docs/spfx/library-component-tutorial.md

```diff
@@ -55,15 +55,24 @@ localization_priority: Priority
 1. Run `gulp` on the command prompt to see everything builds fine.
 
 ## How to consume a 3rd party SPFx library (for local testing)
-1. Run `npm link` from the root directory of library solution. In this case it would be from the **```corporate-library```** folder.
+1. Create a web part project in a **separate project folder, so not in the library project folder structure**, following the instructions from [here](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/web-parts/get-started/build-a-hello-world-web-part#to-create-a-new-web-part-project). Name your web part 'CorporateWebPart'. Like the library before the web part also need to target to ***SharePoint Online only (latest)**.
  
-1. This will create a local npm link to the library with the name which is provided in the ```package.json```.
+1. From the root of the new web part folder, run the command ```npm install <path to library component>```. In this case, the command is ```npm install ..\corporate-library --save-dev```
  
-1. Create a web part project in a **separate project folder, so not in the library project folder structure**, following the instructions from [here](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/web-parts/get-started/build-a-hello-world-web-part#to-create-a-new-web-part-project). Name your web part 'CorporateWebPart'
- 
-1. From the root of the new web part folder, run the command ```npm link corporate-library```
- 
-1. This will create a symbolic link to that locally built library in to the web part and will make it available to your web part.
+1. A symbolic link to the library component project gets created in your `node_modules` folder and adds the following entry in the **package.json** file.
+
+    ```
+	  "devDependencies": {
+      ...
+      "ajv": "~5.2.2",
+      "corporate-library": "file:../corporate-library",
+      "gulp": "~3.9.1"
+      ...
+    }
+    ```
+
+> [!NOTE]
+> This procedure work when the **corporate-library** and **corporate-webpart** exist in the same repository. ``npm install``` install offers more options to integrate library components like this. Git and other Repositories are also an option. For more information, look at the [npm install documentation](https://docs.npmjs.com/cli/install).
  
 1. Open the web part solution in your preferred editor and navigate to `src\webparts\corporateWebPart\CorporateWebPartWebPart.ts`
```

Comment:
Use bold formatting for file paths & names

Reply:
Updated to bold.